### PR TITLE
add sygma to L2Beat `Bridges` dashboard

### PR DESCRIPTION
**Pull request by haochizzle**
adding sygma interoperabillity protocol to L2Beat dashboard:
- [GitHub](https://github.com/sygmaprotocol)
- [Explorer](https://scan.buildwithsygma.com)
- [Website](https://buildwithsygma.com)
- [Docs](https://docs.buildwithsygma.com)

will reach out on discord as well!

### CODE REVIEW

File: packages/config/src/bridges/sygma.ts [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+export const sygma: Bridge = {
+  type: 'bridge',
+  id: ProjectId('sygma'),
+  display: {
+    name: 'Sygma',
+    slug: 'sygma',
+    category: 'Hybrid',
+    description:
+      'Sygma is a modular, open-source, cross-consensus interoperability protocol enabling asset transfers, non-fungible tokens, and decentralized finance. With Sygma, developers can easily extend their applications across Ethereum mainnet, Base, Cronos, Polygon, Gnosis, Polkadot, Kusama, and other Substrate-based chains with active work on Bitcoin and Cosmos SDK interoperability.',
```
Comment (alexmmueller) [inline packages/config/src/bridges/sygma.ts:18]: You are talking about two different things here: Types of data (Tokens) and use cases. I suggest to focus on one: ...enabling transfers of fungible and non-fungible assets, general message passing (GMP/AMB) as well as cross-chain execution'

File: packages/config/src/bridges/sygma.ts [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        'https://faucet-ui-stage.buildwithsygma.com/',
+        'https://validator.faucet.chainsafe.dev/upload',
+      ]
+    },
+  },
+  riskView: {
+    validatedBy: {
+      value: 'Third party',
+      description:
+        '4/6 of the MPC group (the Sygma relayer network) is required to create a cross-chain message with the MPC signature.', // sygma eng team currently developing a zk methodology for block header oracle verification which we will implement on mainnet relayers soon. additionally, optimistic approach coming as well. we call this "tailored security"
```
Comment (alexmmueller) [inline packages/config/src/bridges/sygma.ts:51]: Mainnet has seven relayers, pls check: https://www.notion.so/chainsafe/Relayers-BridgeOps-Dashboard-de7d5441a5dd4fe08c5f22574fc57f7e

File: packages/config/src/bridges/sygma.ts [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+      'Khala',
+      'Polygon',
+      'Cronos',
+      'Base',
+      'Gnosis',
+    ],
+    principleOfOperation: {
+      name: 'Principle of operation',
+      description:
+        `Sygma uses verification systems that have been architected to provide the most optionality in order to reach cross-chain consensus. The tailored security verification systems using multi-party computation (MPC), Spectre (ZK), and Zipline (Optimistic) balances degrees of trust minimization and can be combined to ensure the highest level of security.`,
```
Comment (alexmmueller) [inline packages/config/src/bridges/sygma.ts:74]: Zipline and Spectre are not yet in production, not sure if we here are suggesting too much. Thoughts?

File: packages/config/src/bridges/sygma.ts [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+        {
+          address: EthereumAddress(
+            '0x197C57440A30cB28103ab27CB1b0dC86E5907ADA', 
+          ),
+          type: 'EOA', 
+        },
+      ],
+      name: 'Admin MultiSig Participants',
+      description:
+        'Participants of the 3/5 Sygma admin multisig.',
```
Comment (alexmmueller) [inline packages/config/src/bridges/sygma.ts:262]: I wonder if it's a good step on transparency if we add here the link to the docs page where folks can see WHO actually controls the MultiSig?

**Review by alexmmueller — CHANGES_REQUESTED** [review]
Great job! Left some feedback I think we should consider.

**Comment by mateuszradomski** [comment]
Hi,

Is this PR ready for us to review? I'm confused by the review left by @alexmmueller. It seems like there are somethings that you need to agree on between each other before creating a pull request like that. Either close the PR or mark it as draft and only when you're happy with the state of it move it to ready for review.

Please also take a look at [CONTRIBUTING.md](https://github.com/l2beat/l2beat/blob/main/CONTRIBUTING.md) to make sure that our CI passes on your PR.

**Comment by alexmmueller** [comment]
Thank you @mateuszradomski! Let me fix it.

File: packages/config/src/bridges/sygma.ts (hunk 2 above)
Comment (haochizzle) [reply packages/config/src/bridges/sygma.ts:51]: thanks, was using info from nikola. updated.

File: packages/config/src/bridges/sygma.ts (hunk 1 above)
Comment (haochizzle) [reply packages/config/src/bridges/sygma.ts]: nabbed from the whitepaper, but i can modify per your suggestion. updated.

File: packages/config/src/bridges/sygma.ts (hunk 4 above)
Comment (haochizzle) [reply packages/config/src/bridges/sygma.ts:262]: it's included at the bottom under `knowledgeNuggets`

File: packages/config/src/bridges/sygma.ts (hunk 3 above)
Comment (haochizzle) [reply packages/config/src/bridges/sygma.ts:74]: the MPC verification system _is_ in production. however, the tailored security system is a foundational feature of our protocol so it is still worth mentioning even if those components are still only in testnet. is this still viable @lucadonnoh ?

Comment (lucadonnoh) [reply packages/config/src/bridges/sygma.ts:74]: no, we don’t want to mention things that are not on production. once they are they will be mentioned

Comment (haochizzle) [reply packages/config/src/bridges/sygma.ts]: fair fair. updated again :)

**Comment by mateuszradomski** [comment]
Hi, please, pretty please make sure that _the CI is passing_. Everything you need to know to make it pass is described in [CONTRIBUTING.md](https://github.com/l2beat/l2beat/blob/main/CONTRIBUTING.md). If you have any problems please jump by our public Discord, we'll try to help you as much we can.

**Comment by mateuszradomski** [comment]
Took the liberty to fix some of the whoopsies that happened in previous commits.

Now please hold on tight, someone from the research team will review this PR when we find the capacity.

**Comment by haochizzle** [comment]
Much appreciated @mateuszradomski 🙏🏾

**Pull request closed without merging**